python/cudf/cudf/tests/test_categorical.py

```diff
@@ -401,8 +414,9 @@ def test_categorical_as_unordered(pd_str_cat, inplace):
     [
         pytest.param(
             True,
-            marks=pytest.mark.xfail(
-                reason="https://github.com/pandas-dev/pandas/issues/43232"
+            marks=pytest.mark.skipif(
```

Comment:
This `xfail` can be replaced by a `skipif` because the underlying bug https://github.com/pandas-dev/pandas/issues/43232 was resolved in pandas 1.3.4.